cloud/pkg/taskmanager/manager/executor.go

```diff
@@ -210,8 +210,18 @@ func initExecutor(message util.TaskMessage) (*Executor, error) {
 	if len(nodeStatus) == 0 {
 		nodeList := controller.ValidateNode(message)
 		if len(nodeList) == 0 {
+			nodes, _ := controller.GetTaskNodes(message.Name)
+			nodeStatus = make([]v1alpha1.TaskStatus, len(nodes))
+			for i, nodeName := range nodes {
+				nodeStatus[i] = v1alpha1.TaskStatus{NodeName: nodeName, State: api.TaskFailed}
+			}
+			err = controller.UpdateNodeStatus(message.Name, nodeStatus)
+			if err != nil {
+				return nil, err
+			}
 			return nil, fmt.Errorf("no node need to be upgrade")
```

Comment:
You can consider the better way to respond to error messages

Reply:
Done